app/serializers/topic_post_count_serializer.rb

```diff
@@ -34,4 +35,16 @@ def primary_group_flair_color
     object[:user]&.primary_group&.flair_color
   end
 
+  def admin
```

Comment:
Can we update this to the pattern where the attributes are not included unless `true` to save space in the JSON?

Can be done by using `true` as the method value then checking the object in the `include_xyz?` method.